shared/fs/row/path-item-info.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import * as React from 'react'
+import {globalStyles, globalColors, isMobile} from '../../styles'
+import {Box, Text} from '../../common-adapters'
+import {formatTimeForFS} from '../../util/timestamp'
+
+type Props = {
+  lastModifiedTimestamp: number,
+  lastWriter: string,
+  wrap?: boolean,
+}
+
+const PathItemInfo = ({lastModifiedTimestamp, lastWriter, wrap}: Props) => (
+  <Box style={wrap ? timeWriterBoxStyleWithWrap : timeWriterBoxStyle}>
+    <Text type="BodySmall" lineClamp={isMobile ? 1 : undefined}>
+      {formatTimeForFS(lastModifiedTimestamp)}
+    </Text>
+    {lastWriter ? (
+      <Text type="BodySmall" style={writerTextStyle} lineClamp={isMobile ? 1 : undefined}>
+        &nbsp;by&nbsp;
```

Comment:
i think you can actually just do 
```
{' by '}
```

Reply:
I tried this but somehow one of the spaces was omitted by react. Will try again.

Reply:
it has to be inside the {}

Reply:
Just tried it again. Only the space after "by" if effective. The leading space gets removed. I guess we'll have to keep those `&nbsp`s?

Reply:
try {' '}by{' '}

Reply:
Sorry just saw this; I just tried `{' '}by{' '}` and it omits the leading space as well.

Reply:
oh, this is due to the dom, yeah you have to use nbsp for the leading space annoyingly